### PR TITLE
Uses effect and ST pervasively

### PR DESCRIPTION
**Pull request by mikesol**

### CODE REVIEW

File: examples/insert/Insert.purs [hunk 1]
```diff
@@ -11,16 +11,16 @@ import Deku.Core (insert, remove)
...
-interval :: Int -> AnEvent Zora Instant
-interval = fromEvent <<< FRP.Event.Time.interval
 
-delay :: forall a. Int -> AnEvent Zora a -> AnEvent Zora a
-delay n = fromEvent <<< FRP.Event.delay n <<< toEvent
+interval :: Int -> Event Instant
+interval = FRP.Event.Time.interval
+
+delay :: forall a. Int -> Event a -> Event a
+delay n = FRP.Event.delay n
```
Comment (PureFunctor) [inline examples/insert/Insert.purs:23]: We could probably just import `interval` and `delay` directly now...

File: examples/nested/Nested.purs [hunk 2]
```diff
@@ -19,24 +19,23 @@ import Deku.Toplevel (runInBodyA)
...
 
-interval :: Int -> AnEvent Zora Instant
-interval = fromEvent <<< FRP.Event.Time.interval
+interval :: Int -> Event Instant
+interval = FRP.Event.Time.interval
 
-delay :: forall a. Int -> AnEvent Zora a -> AnEvent Zora a
-delay n = fromEvent <<< FRP.Event.delay n <<< toEvent
+delay :: forall a. Int -> Event a -> Event a
+delay n = FRP.Event.delay n
```
Comment (PureFunctor) [inline examples/nested/Nested.purs:31]: Same here...

File: src/Deku/Core.purs [hunk 3]
```diff
@@ -231,7 +203,7 @@ type SendToPos =
 derive instance Newtype (DOMInterpret payload) _
 
 newtype DOMInterpret payload = DOMInterpret
-  { ids :: Zora String
+  { ids :: ST Global String
```
Comment (PureFunctor) [inline src/Deku/Core.purs:206]: This is made `ST Global` to accommodate for SSR, right?

Comment (mikesol) [reply src/Deku/Core.purs:206]: Yup!

**Pull request merged**